src/Package/Impl/Sql/Publish/SProcGenerator.cs

```diff
@@ -67,7 +67,7 @@ internal sealed class SProcGenerator {
         private void CreateRCodeTable(SqlSProcPublishSettings settings, EnvDTE.Project targetProject, string targetFolder, EnvDTE.ProjectItem targetProjectItem) {
             var creatTableScriptFile = Path.Combine(targetFolder, CreateRCodeTableScriptName);
             using (var sw = new StreamWriter(creatTableScriptFile)) {
-                sw.WriteLine(Invariant($"CREATE TABLE {settings.TableName}"));
+                sw.WriteLine(Invariant($"CREATE TABLE [{settings.TableName}]"));
```

Comment:
A note: square brackets surrounding the table name is a SQL Server convention to allow using reserved words as table names.